src/vs/workbench/contrib/tasks/common/problemMatcher.ts

```diff
@@ -360,17 +360,23 @@ abstract class AbstractLineMatcher implements ILineMatcher {
 	}
 
 	private parseLocationInfo(value: string): ILocation | null {
-		if (!value || !value.match(/(\d+|\d+,\d+|\d+,\d+,\d+,\d+)/)) {
+		if (!value || !value.match(/(\d+|\d+,\d+|\d+,\d+-\d+|\d+,\d+,\d+,de\d+)/)) {
```

Comment:
Canonical Errors also supports line ranges of the form `<startline>-<endline>`. Do we want that adding too?

Reply:
I've noticed I properly messed up this regex (some odd `,de` added at the end. I'll fix it and add support for line ranges too after work.

Reply:
Fixed this regex now.